libs/telemetry/protos/enum.go

```diff
@@ -24,3 +24,10 @@ const (
 	BundleDeployArtifactPathTypeWorkspace   BundleDeployArtifactPathType = "WORKSPACE_FILE_SYSTEM"
 	BundleDeployArtifactPathTypeVolume      BundleDeployArtifactPathType = "UC_VOLUME"
 )
+
+const (
```

Comment:
These are not enums in the proto. I recommend moving them to `bundle/metrics` for ex, to disambiguate what's in the protos from what we define ourselves.